tests/test_zenhub.py

```diff
@@ -46,31 +50,29 @@ def json(self):
         assert False
 
 
-class ZenHub_with_mocked_token(ZenHub):
-    def __init__(self, repo_name=None, issue=None):
+class ZenHubWithMockedToken(ZenHub):
+    def __init__(self, repo_name, issue):
         self.access_params = {'options': 'https://api.zenhub.io/p1/repositories', 'api_token': 99999999}
         self.repo_name = repo_name
-        self.repo_id = self._get_repo_id(repo_name)
+        self.repo_id = 123456789
         self.issue = str(issue)
         self.url = self._generate_url()
 
 
 class TestZenHub(unittest.TestCase):
```

Comment:
Maybe add some more comments explaining what the tests are doing? This section is pretty confusing to me, but I'm also not familiar with mocking

Reply:
I removed the auxiliary class (which I used to construct a specific instance). This is now done with patching and hopefully less confusing. Otherwise, the titles of the tests should indicate the purpose of it. If not please suggest better titles.